pallets/pools/src/tests.rs

```diff
@@ -650,6 +660,15 @@ fn epoch() {
 			pool.reserve.total + senior_epoch.token_price.saturating_mul_int(250 * CURRENCY),
 			1010 * CURRENCY
 		);
+
+		assert!(
+			<Pools as PoolInspect<
+				<Test as frame_system::Config>::AccountId,
+				<Test as Config>::CurrencyId,
+			>>::get_tranche_token_price(0, SeniorTrancheId::get())
```

Comment:
Maybe we can extract this to a test utils function to reduce noise

Reply:
I think this is too little to actually make sense to extract (it's just a function call). We definitely should extract a lot of other things in the pools tests to separate utils, but I'd consider that for a future PR.

Reply:
https://github.com/centrifuge/centrifuge-chain/issues/885

Reply:
I disagree. Sure there are bigger fish to fry but this takes 1 minute to implement; These minor but noisy implements easily creep up throughout the codebase and it's completely unnecessary.

Reply:
My point isn't that I don't want to spend time on changing this here (I'm just pointing to larger refactors as something to do later). It's that I don't think this should be moved into a separate test util at all.

This is already a single function call, if we start moving all of those into other separate functions, we just bloat the test code with utils.

In my opinion, a heuristic for whether something should be moved into a helper is that both of these apply:
- The same code is called multiple times => this is the case here.
- The code at least combines >1 function calls or otherwise has some complexity => this isn't the case here.

Reply:
Thanks for explaining where you stand on this. I find this approach inconsistent with, for example, how we have type aliases in our pallets (`AccountOf<T>`, `OriginOf<T>`, etc); it's all about reduce and make what's important easier to read/look at. 

In this case, if we could do `assert_eq(get_tranche_token_price(a, b), x)` or `assert_eq(PoolInspectOf<Pools>::get_tranche_token_price(a, b), x)` instead of of having all of this noisy lines of code duplicating type casting and references, that'd be a plus.

Leaving this one here tho 👍

Reply:
I think it's worth adding the type alias or a one-line helper, but not strongly enough to block this on it :shrug:

Reply:
I don't really agree with the one-line helper for a single function (since this would be adding a one-line helper for a one-line function, and if we do that consistently, we'll have a huge amount of helpers), but I hadn't considered the option of the type alias (we're not actually using any of these in the tests right now). I think that makes more sense. I'll still add this in another PR.